### ISSUE TITLE
ArchLinux does not use the specified paths

### ISSUE THREAD

**Issue by Merith-TK** [cdr/sshcode#133]
archlinux, when installing from the official repo atleast with `pacman -Sy code` uses `$HOME/.vscode-oss` and `$HOME/.config/Code - OSS`

Just wanted to state this

**Comment by peschu123** [comment]
Thank you Merith, I was wondering, why my extensions are not loaded. I use Manjaro as desktop(local) and archlinux on the server(remote).

### the following steps worked for me as a quickfix/workaround
1.) delete the wrong/empty folders
***!!!! danger zone !!!!***(be careful with -Rf >> (f)orce (R)ecursive delete)
```bash
rm -Rf $HOME/.vscode
rm -Rf $HOME/.config/Code

```
2.) a symlink to "fix" it:
```bash
ln -s $HOME/.vscode-oss $HOME/.vscode
ln -s "$HOME/.config/Code - OSS" $HOME/.config/Code
```
I hope that helps somebody ...

Would be good to have at least an option to set the "source" folder if it is not automatically determined or some env vars 

Thanks @all for the good work, code-server is awesome =)

**Comment by Merith-TK** [comment]
Agreed! i love it as it makes things so much easier to do when my main machine (windows) breaks itself cuz updates, i can just ssh to my server with my workspace and do shit there
